code/modules/mob/living/living_say.dm

```diff
@@ -213,6 +213,11 @@ GLOBAL_LIST_EMPTY(channel_to_radio_key)
 		var/msg
 		if(!first_piece.speaking || !(first_piece.speaking.flags & NO_TALK_MSG))
 			msg = "<span class='notice'>[src] talks into [used_radios[1]]</span>"
+			var/list/special_radio_channels = list("Syndicate", "SyndTeam", "Security", "Procedure", "Command", "Response Team", "Special Ops")
```

Comment:
I think this should be slightly better for performance, especially since we don't modify it
```suggestion
			var/static/list/special_radio_channels = list("Syndicate", "SyndTeam", "Security", "Procedure", "Command", "Response Team", "Special Ops")
```